src/app/core.ts

```diff
@@ -84,6 +84,16 @@ export interface AppOptions {
   httpAgent?: Agent;
 }
 
+/**
+ * A Firebase app holds the initialization information for a collection of
+ * services.
+ *
+ * Do not call this constructor directly. Instead, use
+ * {@link
+ *   https://firebase.google.com/docs/reference/admin/node/admin#.initializeApp
+ *   `admin.initializeApp()`}
```

Comment:
Should this be just `initializeApp()`, and below?

Reply:
This API is shared between both old and modularized APIs. So we can go either way here. For now I'm going to keep the old documentation here.